perf-harness/src/test/java/io/opentelemetry/perf/OtlpPipelineStressTest.java

```diff
@@ -243,16 +245,22 @@ private static IntervalMetricReader setupSdk(MetricExporter metricExporter) {
 
     // set up the metric exporter and wire it into the SDK and a timed reader.
 
-    return IntervalMetricReader.builder()
-        .setMetricExporter(metricExporter)
-        .setMetricProducers(
-            Collections.singleton(
-                ((SdkMeterProvider) GlobalMetricsProvider.get()).getMetricProducer()))
-        .setExportIntervalMillis(1000)
-        .build();
-  }
+    Resource resource =
+        Resource.create(
+            Attributes.builder()
+                .put(ResourceAttributes.SERVICE_NAME, "PerfTester")
```

Comment:
heh...this is more of a "how does the SDK react to bad situations" than perf per se, but this is fine. :)

Reply:
And, I had put `perf` in the name of the tracer, so I'm mostly to blame here. ;)

Reply:
Oops forgot to remove the unnecessary system property too, phew